common/io/include/claragenomics/io/fasta_parser.hpp

```diff
@@ -59,9 +59,10 @@ class FastaParser
 ///
 /// \param fasta_file Path to FASTA(.gz) file. If .gz, it must be zipped with bgzip.
 /// \param min_sequence_length Minimum length a sequence needs to be to be parsed. Shorter sequences are ignored.
+/// \param shuffle Enables shuffling reads
 ///
 /// \return A unique pointer to a constructed parser object.
-std::unique_ptr<FastaParser> create_kseq_fasta_parser(const std::string& fasta_file, int min_sequence_length = 0);
+std::unique_ptr<FastaParser> create_kseq_fasta_parser(const std::string& fasta_file, int min_sequence_length = 0, bool shuffle = true);
```

Comment:
@r-mafi since this change isn't directly used by this PR, let's create a separate PR with this `shuffle` option so cudamapper folks can review it

Reply:
since it was minor without any impact on the current code (...,`bool shuffle = true`), I found it harmless to be here. But OK, will submit in a new PR and ask Mike to review.

Reply:
done.